src/Topography/TopographyFileRenderer.cpp

```diff
@@ -159,7 +159,7 @@ TopographyFileRenderer::Paint(Canvas &canvas,
 
   UpdateArrayBuffer();
   array_buffer->Bind();
-  const ShapePoint *const buffer = nullptr;
+  const ShapePoint *const buffer = 0;
```

Comment:
That's wrong

Reply:
What i was trying to fix here, is its first use in whats otherwise a nullptr addition:
https://github.com/XCSoar/XCSoar/blob/a7bffc83982970d2cc2d204803ec5b576ffc15b8/src/Topography/TopographyFileRenderer.cpp#L205

Reply:
1. your change doesn't "fix" this, it's still a null pointer, just not using the `nullptr` keyword, but the `0` constant which is effectively the same here
2. not a bug; this isn't an actual pointer, but an offset into an OpenGL buffer (`GL_ARRAY_BUFFER`), see https://registry.khronos.org/OpenGL-Refpages/es3.0/html/glVertexAttribPointer.xhtml